src/commands/lint/lint.s3.paths.ts

```diff
@@ -0,0 +1,70 @@
+import { command, positional, string } from 'cmd-ts';
+
+import { CliInfo } from '../../cli.info.js';
+import { logger } from '../../log.js';
+import { config, registerCli, verbose } from '../common.js';
+import { imageryBucketNames, imageryCrs, imageryProducts, regions } from './lint.constants.js';
+
+export const commandLintInputs = command({
+  name: 'lint-inputs',
+  description: 'Pretty print JSON files',
+  version: CliInfo.version,
+  args: {
+    config,
+    verbose,
+    path: positional({
+      type: string,
+      displayName: 'path',
+      description: 'JSON file to load inputs from, must be a JSON Array',
+    }),
+  },
+
+  async handler(args) {
+    registerCli(this, args);
+    const startTime = performance.now();
+
+    logger.info('LintInputs:Start');
+    if (!args.path) {
+      throw new Error('No path Provided');
+    }
+    logger.info({ target: args.path }, 'LintInputs:Info');
+    lintPath(args.path);
+    logger.info({ duration: performance.now() - startTime }, 'LintInputs:Done');
```

Comment:
Shouldn't runtime and the like be lower level than info? I think it's usually debug level.

Reply:
the `info` is more that the command finished ok, it is just dumping more helpful info at the same time.